src/pool/srv_iv.c

```diff
@@ -258,6 +275,20 @@ pool_iv_prop_g2l(struct pool_iv_prop *iv_prop, daos_prop_t *prop)
 				prop_entry->dpe_val_ptr = dst_list;
 			}
 			break;
+		case DAOS_PROP_PO_POLICY:
+			iv_prop->pip_policy_desc = (void *)
+				(iv_prop->pip_iv_buf +
+				roundup(iv_prop->pip_policy_desc_offset, 8));
+			pd = iv_prop->pip_policy_desc;
+
+			D_ALLOC(pd_alloc, sizeof(*pd));
+			if (pd_alloc != NULL) {
+				memcpy(pd_alloc, pd, sizeof(*pd));
+				prop_entry->dpe_val_ptr = pd_alloc;
+			} else {
+				D_GOTO(out, rc = -DER_NOMEM);
+			}
```

Comment:
You get less indentation if you just do a conditional goto rather than using else.

```suggestion
			D_ALLOC(pd_alloc, sizeof(*pd));
			if (pd_alloc == NULL)
				D_GOTO(out, rc = -DER_NOMEM);

			memcpy(pd_alloc, pd, sizeof(*pd));
			prop_entry->dpe_val_ptr = pd_alloc;
```